d8/d8batch/config/install/field.storage.node.field_feed_source_url.yml

```diff
@@ -0,0 +1,19 @@
+uuid: 6d09794e-a019-42ae-9a43-67df9009c074
+langcode: en
```

Comment:
Also a good question, my answer is the same as above, these file also came from Drupal CMI UI directly. But I'll try to investigate the reason.

![screen shot 2015-11-17 at 07 44 14](https://cloud.githubusercontent.com/assets/1755573/11204429/1480096c-8cff-11e5-95fc-4e498e2b2d4c.png)